AppCenter/AppCenter/MSAppCenter.m

```diff
@@ -106,6 +106,15 @@ + (BOOL)isConfigured {
   return [MSAppCenter sharedInstance].sdkConfigured && [MSAppCenter sharedInstance].configuredFromApplication;
 }
 
++ (BOOL)isRunningInAppCenterTestCloud {
```

Comment:
I think it's worth adding a small unit test for this. There is a similar unit test [here](https://github.com/microsoft/appcenter-sdk-apple/blob/27b6cdf2c9576ae3655b63488abaa6e9f511f476/AppCenter/AppCenterTests/MSAppCenterTests.m#L455).

Reply:
Good call @clpolet - test added!